make/modules/java.base/Lib.gmk

```diff
@@ -233,3 +233,25 @@ ifeq ($(ENABLE_FALLBACK_LINKER), true)
 
   TARGETS += $(BUILD_LIBFALLBACKLINKER)
 endif
+
+################################################################################
+
+ifeq ($(call isTargetOs, linux)+$(call isTargetCpu, x86_64)+$(INCLUDE_COMPILER2), true+true+true)
+  ifeq ($(TOOLCHAIN_TYPE), gcc)
```

Comment:
This requirement can be folded into the "and" sequence in the `ifeq` statement above. There is nothing that really merits it to be specified separately.

Reply:
Hello Magnus (@magicus),

As suggested, made the modification to use a single `ifeq` (which works for me on a Linux machine). Could you please confirm if the approach below is correct? 

`ifeq ($(call isTargetOs, linux)+$(call isTargetCpu, x86_64)+$(INCLUDE_COMPILER2)+$(filter $(TOOLCHAIN_TYPE), gcc), true+true+true+gcc)`

Thanks,
Vamsi